src/components/EventSuggeestionButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { SettingsContext } from '../SettingsProvider';
+
+import { Button } from './Button';
+import { EVENT_SUGGESTION_BUTTON } from './screens';
+
+export const EventSuggestionButton = ({
+  buttonTitle,
+  onPress
+}: {
+  buttonTitle: string;
+  onPress: () => void;
+}) => {
+  const { globalSettings } = useContext(SettingsContext);
+  const { navigation: navigationType, sections = {} } = globalSettings;
+  const { eventListIntro } = sections;
+
+  return (
+    <View
+      style={
+        eventListIntro.buttonType === EVENT_SUGGESTION_BUTTON.BOTTOM_FLOATING && [
+          styles.floatingButtonContainer,
+          stylesWithProps({ navigationType }).position
+        ]
+      }
+    >
+      <Button
+        onPress={onPress}
+        title={buttonTitle}
+        notFullWidth={eventListIntro.buttonType === EVENT_SUGGESTION_BUTTON.BOTTOM_FLOATING}
+      />
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  floatingButtonContainer: {
+    alignSelf: 'center',
+    position: 'absolute'
+  }
+});
+
+/* eslint-disable react-native/no-unused-styles */
+/* this works properly, we do not want that warning */
+const stylesWithProps = ({ navigationType }: { navigationType: string }) => {
+  return StyleSheet.create({
+    position: {
+      bottom: navigationType === 'drawer' ? '5%' : 0
+    }
+  });
+};
```

Comment:
i cannot overscroll the button to see the last entry in the list.
compared to pois with floating button, where i can overscroll to see the bottom last entry of the list entirely

|events|pois|
|---|---|
|![Simulator Screen Recording - iPhone 16 - 2024-12-03 at 15 59 38](https://github.com/user-attachments/assets/d7d2abb0-4ac3-49b8-b4fc-e7ddb56d62d8)|![Simulator Screen Recording - iPhone 16 - 2024-12-03 at 16 02 47](https://github.com/user-attachments/assets/5d75cf32-29af-4652-8d7e-a6bcf37e3cbc)|

please add the "padding" after the list as well for events, if floating button is set

Reply:
I actually added a padding for it, but it doesn't seem to be working correctly. I will update it.